egs/swbd/s5c/local/reverberate/rir_generator.cpp

```diff
@@ -0,0 +1,459 @@
+/*
```

Comment:
Please do not  include external c++ code like this. I think it would be better to add this to external tools folder in kaldi/tools/extra, add the appropriate **download** and installation script. Check for the binary in the local script and if it is not found print a user message asking the user to run the installation.